recipes/leftbar/contact_row/contact_row.vue

```diff
@@ -248,6 +248,12 @@ export default {
     avatarInitial () {
       return this.name?.[0] ?? '';
     },
+
+    contactDescription () {
+      return [this.name, this.presenceText, this.userStatus]
+        .filter(str => !!str)
```

Comment:
Since we are using this 

`.filter(str => !!str).join(' ');`
in 
recipes/leftbar/contact_row/contact_row.vue
recipes/leftbar/general_row/general_row.vue
recipes/leftbar/group_row/group_row.vue

Could we create a util function in leftbat to handle it?

Reply:
Sounds like a good utility, I'll create it.